CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - `Security` in case of vulnerabilities.
 
 ## [x.x.x] - unreleased
+### Added
```

Comment:
Let's tag as `Changed` we change the type of return exception making it more clear

Reply:
done